src/cpp/session/SessionModuleContext.cpp

```diff
@@ -348,6 +348,17 @@ SEXP rs_rstudioEdition()
 
 // get version
 SEXP rs_rstudioVersion()
+{
+   std::string processedVersion = RSTUDIO_VERSION;
+   processedVersion = boost::regex_replace(processedVersion, boost::regex("(?:-[a-zA-Z]*)?\\+"), ".");
```

Comment:
As a habit, I think it's usually better to hoist the `boost::regex()` object into its own variable, since (at least for other regex functions) the regular expression usually needs to live at least as long as other objects derived from matching it (e.g. when finding and generating matches).

It's probably fine here, but pointing this out just in case.

Reply:
I didn't know that! Is that primarily for when you'll iterate through multiple matches in a single input?

At any rate, I agree that it's necessary here since we keep no objects except the resulting string, which can't have a dependence on anything in boost.